custom-jest-env.js

```diff
@@ -0,0 +1,13 @@
+const Environment = require("jest-environment-jsdom");
+
+module.exports = class CustomTestEnvironment extends Environment {
```

Comment:
you do need to start a node right now.
I've experimented with starting the node inside a jest test, but the test would timeout or I would have memory leaks and the port the node runs on would be blocked for other tests to use. 
It will research more on how to do this when I get the chance / need to write more tests